### ISSUE TITLE
new weapon

### ISSUE THREAD

**Issue by fhgb54643689** [studio-minus/ppg-bugs-and-requests#171]
**Feature description**
a gas gun basically a flamethrower which shoots out syringe gas in which you would connect a blood vessel wire from said syringe to the device and it would shoot out a flamethrower like effect of that syringe type

**Feature purpose**
mainly because its kind of difficult to have say a chamber which automatically makes a human alive again with syringes because of the wacky physics sometimes

**Additional context**
none really

**Comment by BBQGiraffe** [comment]
I feel like having a little slot where you could put in a syringe would look better, as seen in my expertly drawn design 
![godfearsme](https://user-images.githubusercontent.com/45021140/131858640-d2a64d9e-5799-40fa-8f82-65188c5700f6.png)

**Comment by fhgb54643689** [comment]
> I feel like having a little slot where you could put in a syringe would look better, as seen in my expertly drawn design
> ![godfearsme](https://user-images.githubusercontent.com/45021140/131858640-d2a64d9e-5799-40fa-8f82-65188c5700f6.png)

yeah agreed

**Comment by mestiez** [comment]
the problem with the slot thing is that it would limit the apparent use of the weapon. it should have a generic looking inlet at the top that indicates that any liquid container can be attached

**Comment by fhgb54643689** [comment]
> the problem with the slot thing is that it would limit the apparent use of the weapon. it should have a generic looking inlet at the top that indicates that any liquid container can be attached

honestly i would not mind either way as long as it still works

**Comment by fhgb54643689** [comment]
> the problem with the slot thing is that it would limit the apparent use of the weapon. it should have a generic looking inlet at the top that indicates that any liquid container can be attached

so do you think something like this would get implemented at some point?

**Comment by BBQGiraffe** [comment]
> the problem with the slot thing is that it would limit the apparent use of the weapon. it should have a generic looking inlet at the top that indicates that any liquid container can be attached

true, it's the ppg community if the weapon looked like this they'd still have no idea what to do with it
![131858640-d2a64d9e-5799-40fa-8f82-65188c5700f6](https://user-images.githubusercontent.com/45021140/131925207-70219e0e-ea36-45ba-832c-8ab496c13495.png)

**Comment by JohnVcav** [comment]
for it to be more obvious I believe it could have a central see-through "canister" and a spraying hose connected to it
![image](https://user-images.githubusercontent.com/89386839/132085460-4da3c716-66f5-4461-9040-962c210d1ff9.png)
or it could be something similar to a paint sprayer, which would probably look/feel the best. (maybe a porthole in the back too?)
![image](https://user-images.githubusercontent.com/89386839/132085530-6fff357f-f6d0-406c-9c90-42d4a38f7cd1.png)

**Comment by roscosthebalm** [comment]
but what if it causes a shit ton of lag

**Comment by JohnVcav** [comment]
> but what if it causes a shit ton of lag

hopefully it would be coded in a way that doesn't cause a shit ton of lag

**Comment by Brostrod** [comment]
This is now more viable I think since attachments exist and detail mode includes where attachments go (the blue dots)
